test_sets/Ideas_for_SDL_must_send_WARNINGS_success_true_to_mobile_app_if_HMI_respond_WARNINGS_at_least_to_one_component_of_RPC.txt

```diff
@@ -0,0 +1,29 @@
+./test_scripts/API/AddCommand/ATF_AddCommand_UI_AddCommand_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_UI_AddCommand_VR_AddCommand_SUCCESS
+./test_scripts/API/AddCommand/ATF_AddCommand_VR_AddCommand_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_VR_AddCommand_UI_AddCommand_SUCCESS
+./test_scripts/API/Alert/ATF_Alert_UI_Alert_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_UI_Alert_TTS_Speak_SUCCESS
+./test_scripts/API/Alert/ATF_Alert_TTS_Speak_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_TTS_Speak_UI_Alert_SUCCESS
+./test_scripts/API/AlertManeuver/ATF_AlertManeuver_Navi_AlertManeuver_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_Navi_AlertManeuver_TTS_Speak_SUCCESS
+./test_scripts/API/AlertManeuver/ATF_AlertManeuver_TTS_Speak_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_TTS_Speak_Navi_AlertManeuve_SUCCESS
+./test_scripts/API/PerformInteraction/ATF_PerformInteraction_UI_PerformInteraction_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_UI_PerformInteraction_VR_PerformInteraction_SUCCESS
+./test_scripts/API/PerformInteraction/ATF_PerformInteraction_VR_PerformInteraction_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_VR_PerformInteraction_UI_PerformInteraction_SUCCESS
+./test_scripts/API/PerformAudioPathThru/ATF_PerformAudioPathThru_UI_PerformAudioPathThru_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_UI_PerformAudioPathThru_TTS_Speak_SUCCESS
+./test_scripts/API/PerformAudioPathThru/ATF_PerformAudioPathThru_TTS_Speak_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_TTS_Speak_UI_PerformAudioPathThru_SUCCESS
+
+
+*****Precondition: HMI does NOT respond to Interface.IsReady
+./test_scripts/API/AlertManeuver/ATF_TTS_IsReady_no_resp_AlertManeuver_TTS_Speak_UNSUP_RES.lua Precondition_HMI_does_not_respond_to_TTS_IsReady_mobile_app_sends_RPC_AlertManeuver_and_HMI_responds_to_Navi_AlertManeuver_with_SUCCESS_and_to_TTS_Speak_with_UNSUPPORTED_RESOURCE_success_true_SDL_should_sent_to_mobile_app_UNSUPPORTED_RESOURCE_success_true
+./test_scripts/API/SetGlobalProperties/ATF_TTS_IsReady_no_resp_SetGlobalProperties_TTS_SetGlobalProperties_gets_UNSUP_RES.lua Precondition_HMI_does_not_respond_to_TTS_IsReady_mobile_app_sends_RPC_SetGlobalProperties_and_HMI_responds_to_UI_SetGlobalProperties_with_SUCCESS_and_to_TTS_SetGlobalProperties_with_UNSUPPORTED_RESOURCE_success_true_SDL_should_sent_to_mobile_app_UNSUPPORTED_RESOURCE_success_true

```

Comment:
@vvvakulenko, as CRQ is covering result code WARNINGS, UI_SetGlobalProperties should return WARNINGS.